src/libexpr/eval-settings.hh

```diff
@@ -5,9 +5,19 @@
 
 namespace nix {
 
+class Store;
+
 struct EvalSettings : Config
 {
-    EvalSettings(bool & readOnlyMode);
+    /**
+     * Function used to interpet look path entries in the form
+     * `<scheme>:<arbitrary string>`.
+     */
+    using LookupPathHook = std::optional<std::string>(ref<Store> store, std::string_view);
```

Comment:
Could you document whether the scheme is passed to the hook, or just the part after it?

Reply:
Also, what is the meaning of the return value?